mysql-test/suite/binlog_encryption/t/binlog_encryption_on_while_binlog_off.test

```diff
@@ -0,0 +1,16 @@
+# PS-287: It is possible to enable encrypt_binlog with no binary logging
+#
+# It is possible to enable encrypt_binlog without binary logging. In this setup
+# server will only encrypt relay logs - if used. We print an information in server's log
+# file to notify user that although encrypt_binlog in on, there is no binlog file to
+# encrypt and only relay logs will get encrypted.
+
+--echo # Should be 0
+select @@global.log_bin;
+--echo # Should be 1
+select @@global.encrypt_binlog;
+
+# Find a note in server's log
+--let SEARCH_FILE= $MYSQLTEST_VARDIR/log/mysqld.1.err
```

Comment:
Must restart the server (in the testcase) with a custom error log path, grep that, restart with the default one and delete the custom one in the end. Otherwise --repeat and simply executing this testcase after another one that hit the same condition will be broken

Reply:
Done